ui/component/or-components/src/or-file-uploader.ts

```diff
@@ -0,0 +1,133 @@
+import { css, html, LitElement, PropertyValues } from "lit";
```

Comment:
Copyright

Reply:
* Should just have a simple `get Files(): FileList {}` so it behaves like the standard File input element just with styling added (don't do any encoding and exposing that as a custom event)
* `or-asset-viewer` `fileupload-elem` should also be replaced with this component